src/cls/cephfs/cls_cephfs.cc

```diff
@@ -123,6 +123,70 @@ static int accumulate_inode_metadata(cls_method_context_t hctx,
   return 0;
 }
 
+// I want to select objects that have a name ending 00000000
+// and an xattr (scrub_tag) not equal to a specific value.
+// This is so special case that we can't really pretend it's
+// generic, so just fess up and call this the cephfs filter.
```

Comment:
When we're writing a plugin for the osd, I don't feel the need to make it customizable like this: it's already a special purpose call, and anyone who needs something more generic can write their own object class.

I'm going to take you at the "I think this is fine" part :-)